lib/fastlane/plugin/firebase_app_distribution/helper/firebase_app_distribution_auth_client.rb

```diff
@@ -0,0 +1,48 @@
+require 'fastlane_core/ui/ui'
+module Fastlane
+  UI = FastlaneCore::UI unless Fastlane.const_defined?("UI")
+  module Auth
+    module FirebaseAppDistributionAuthClient
+      TOKEN_CREDENTIAL_URI = "https://oauth2.googleapis.com/token"
+
+      def fetch_auth_token(google_service_path)
```

Comment:
If we have the ability to do Google service credentials, why wouldn't we use that? Is there some reason that must be the order?

Reply:
I'm guessing it's because that's how the Firebase CLI works? I think we'd want it to continue to have the same precedence so nothing changes when folks upgrade.

Reply:
Will change so it reflects the way other systems do it. I don't believe that should change the user experience, but that may be a question to ask once we give it to customers if they have a preference to the order.

Reply:
👍 good call out @mannyjimenez0810 something we can definitely ask customers about 🙂

Reply:
order should be google_service_path -> ENV["GOOGLE_APPLICATION_CREDENTIALS"] -> ENV["FIREBASE_TOKEN"]